app/models/hotel.rb

```diff
@@ -1,3 +1,9 @@
 class Hotel < ApplicationRecord
   has_many :rooms
+  has_many :room_guests, through: :rooms
+  has_many :guests, through: :room_guests
```

Comment:
This association works, but it's actually one more line than we really need... Consider that a Hotel has many Rooms, and a Room has many Guests. Therefore, we could write an association that follows that logic: 

```
has_many :guests, through: :rooms
```

Reply:
this is good to know!

Reply:
```suggestion
  has_many :guests, through: :room_guests
```